module/zstd/zfs_zstd.c

```diff
@@ -781,8 +897,8 @@ module_init(zstd_init);
 module_exit(zstd_fini);
 #endif
 
-EXPORT_SYMBOL(zfs_zstd_compress);
-EXPORT_SYMBOL(zfs_zstd_decompress_level);
-EXPORT_SYMBOL(zfs_zstd_decompress);
-EXPORT_SYMBOL(zfs_zstd_cache_reap_now);
+ZFS_MODULE_PARAM(zfs, zstd_, earlyabort_pass, INT, ZMOD_RW,
+	"Should we try early abort when using zstd?");
```

Comment:
```suggestion
	"Enable early abort attempt when using zstd");
```

Reply:
I'd probably say "Enable early abort attempt**s**", but LGTM; I'll push that, thanks.